specification/authorization/resource-manager/Microsoft.Authorization/stable/2015-07-01/authorization-ClassicAdminCalls.json

```diff
@@ -133,13 +133,6 @@
       "required": true,
       "type": "string",
       "description": "The ID of the target subscription."
-    },
-    "ApiVersionParameter": {
```

Comment:
@dsgouda reason for deleting this?

Reply:
Simply because I couldn't generate the code otherwise